src/System.Security.Cryptography.OpenSsl/pkg/System.Security.Cryptography.OpenSsl.pkgproj

```diff
@@ -4,20 +4,11 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Security.Cryptography.OpenSsl.csproj">
-      <SupportedFramework>netstandardapp1.5</SupportedFramework>
-    </ProjectReference>
-    <ProjectReference Include="..\src\System.Security.Cryptography.OpenSsl.csproj" >
-      <OSGroup>Linux</OSGroup>
+      <SupportedFramework>netstandardapp1.5;net461;netcore50</SupportedFramework>
```

Comment:
That's just saying that we expect to see the netstandard assembly that throw platform-not-supported.